executor/executor.go

```diff
@@ -107,7 +108,7 @@ type baseExecutor struct {
 	retFieldTypes   []*types.FieldType
 }
 
-// Open implements the Executor Open interface.
+// Open opens all executors and initializes childrenResults according to executors' length.
```

Comment:
// Open initializes children recursively and "childrenResults" according to children's schemas.